src/compare/multiply.ts

```diff
@@ -0,0 +1,50 @@
+import { Image } from '../Image';
+import { getOutputImage } from '../utils/getOutputImage';
+import { validateChannels } from '../utils/validators/validators';
+
+export interface OperationBasedOptions {
+  /**
+   * Channels where value will be multiplied.
+   * @default all channels
+   */
+  channels?: number[];
+  /**
+   * Image to which the resulting image has to be put.
+   */
+  out?: Image;
+}
```

Comment:
This is not good.

First the name is not good. Then you use the term "multiply" to document something that is used by multiply and divide.

There are many other methods that use those 2 options. To do this properly, the base should be shared with many other methods (so big refactoring).

So please just use 2 types `MultiplyOptions` and `DivideOptions`. It's ok that they have duplicate properties.